src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Semantics/Symbols/AggregateSymbol.cs

```diff
@@ -109,19 +109,16 @@ public bool FindBaseAgg(AggregateSymbol agg)
             return false;
         }
 
-        public NamespaceOrAggregateSymbol Parent
-        {
-            get { return parent.AsNamespaceOrAggregateSymbol(); }
-        }
+        public NamespaceOrAggregateSymbol Parent => parent as NamespaceOrAggregateSymbol;
 
         public bool isNested()
         {
-            return parent != null && parent.IsAggregateSymbol();
+            return parent is AggregateSymbol;
         }
 
         public AggregateSymbol GetOuterAgg()
```

Comment:
We should stick to one style. Either arrow functions or full functions for these one liners (arrows IMO for functions similar to this).